dpdispatcher/submission.py

```diff
@@ -532,11 +541,10 @@ def handle_unexpected_job_state(self):
                 raise RuntimeError(f"job:{self.job_hash} {self.job_id} failed {self.fail_count} times.job_detail:{self}")
             self.submit_job()
             dlog.info("job:{job_hash} re-submit after terminated; new job_id is {job_id}".format(job_hash=self.job_hash, job_id=self.job_id))
+            time.sleep(0.2)
```

Comment:
What's the reason of this sleep?

Reply:
for unittest usage;  I try to test if the submission can resubmit jobs as expected. and the jobs can be finished within 0.2 seconds. If the program don't sleep. The program may wait for another 30 seconds